tests/1-03-conformance.js

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2022 - 2024 Digital Bazaar, Inc.
+ *
+ * SPDX-License-Identifier: LicenseRef-w3c-3-clause-bsd-license-2008 OR LicenseRef-w3c-test-suite-license-2023
+ */
+
+import {addPerTestMetadata, setupMatrix, trimText} from './helpers.js';
+import {
+  shouldBeSecured,
+  shouldIncludeAllRequiredProperties
+} from './assertions.js';
+import assert from 'node:assert/strict';
+import chai from 'chai';
+import {createRequire} from 'module';
+import {filterByTag} from 'vc-test-suite-implementations';
+import {TestEndpoints} from './TestEndpoints.js';
+
+// eslint-disable-next-line no-unused-vars
+const should = chai.should();
+
+const require = createRequire(import.meta.url);
+
+const tag = 'vc2.0';
+const {match} = filterByTag({tags: [tag]});
+
+// // 1.3 Conformance https://w3c.github.io/vc-data-model/#conformance
+describe('1.03 Conformance', function() {
+  setupMatrix.call(this, match);
+  for(const [name, implementation] of match) {
+    const endpoints = new TestEndpoints({implementation, tag});
+
+    describe(name, function() {
+      let issuedVc;
+      before(async function() {
+        try {
+          issuedVc = await endpoints.issue(require(
+            './input/credential-ok.json'));
+        } catch(e) {
+          console.error(
+            `Issuer: ${name} failed to issue "credential-ok.json".`,
+            e
+          );
+        }
+      });
+      beforeEach(addPerTestMetadata);
+      it(trimText(`Conforming document (compliance): Sections 
+        4. Basic Concepts, 5. Advanced Concepts, and 6. Syntaxes 
```

Comment:
we usually handle this with string concatenation, i.e.

```js
it('some really long ' +
  'some other really long ' +
  'final really long', function() {
})
```
```

Reply:
the goal was to not rely on string concatenation, this removes the need to close/re-open bracket and add a + sign as you can just enter a newline